xml/depl_cisco_opflex.xml

```diff
@@ -427,260 +474,337 @@ horizon
+  </para>
+
+  <figure>
+   <title>Creating a TCP classifier.</title>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="gbp3.png" width="100%" format="png"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="gbp3.png" width="75%" format="png"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+
+  <para>
+   In <menuchoice><guimenu>Policy</guimenu><guimenu>Application 
+Policy</guimenu><guimenu>Policy Rules</guimenu></menuchoice> create two
+   rules, one for TCP and one for ICMP, linking the Policy classifier to the
+   Allow policy action.
+  </para>
+
+  <figure>
+   <title>Linking the Policy classifier to the Allow policy.</title>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="gbp4.png" width="100%" format="png"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="gbp4.png" width="75%" format="png"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+
+  <para>
+   In <menuchoice><guimenu>Policy</guimenu><guimenu>Application Policy 
+</guimenu><guimenu>Policy Rule Set</guimenu></menuchoice>  create
+   a Dbaccess rule set that links to the TCP and ICMP rules.
+  </para>
+
+  <figure>
+   <title>Link Dbaccess to TCP and ICMP rules.</title>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="gbp5.png" width="100%" format="png"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="gbp5.png" width="75%" format="png"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+
+  <para>
+In <menuchoice><guimenu>Policy</guimenu><guimenu>Groups</guimenu></menuchoice> 
+create two internal groups, Web and DB.
+  </para>
+
+  <procedure>
    <step>
     <para>
-     Edit each group so that the DB group is the rule set provider, and the Web group is the rule set consumer.
+     Edit each group so that the DB group is the rule set provider, and the Web
+     group is the rule set consumer.
     </para>
-   </step> 
-   
+   </step>
    <step>
     <para>
-        Click on the Web group and the DB group, and then select <literal>Create Member</literal> to deploy two instances for each group.
+     Click on the Web group and the DB group, and then select <guimenu>Create
+     Member</guimenu> to deploy two instances for each group.
     </para>
    </step>
-   
-      <step>
+   <step>
     <para>
-       Confirm that a Web instance can ping its gateway, the other Web instance and the DB instances.
+     Confirm that a Web instance can ping its gateway, the other Web instance
+     and the DB instances.
     </para>
    </step>
-   
-  <step>
+   <step>
     <para>
-       For further testing, remove the Dbaccess rule set from the Consumed Rule Set for the Wweb group to see the pings stop. Re-add the Dbaccess rule set to see the pings start again.
+     For further testing, remove the Dbaccess rule set from the Consumed Rule
+     Set for the Web group to see the pings stop. Re-add the Dbaccess rule set
```

Comment:
First sentence in this para is rather hard to read... what is the goal of our testing? why is "rule set" capitalized one time and in lowercase the other time? "to see the pings stop" sounds a bit awkward, especially when placed at the end (feels like an afterthought).